### ISSUE TITLE
Realtime GraphQL and serverless made easy for React with Hasura

### ISSUE THREAD

**Issue by shark-h** [graphqlsydney/graphqlsydney#8]
Hasura is a free and open source GraphQL Engine that can help supercharge your GraphQL adoption, whether it is for a new application or for an existing one. The talk will cover:

* Using GraphQL to make CRUD operations from a client
* Setting up access controls to data
* Building realtime components in Vue using GraphQL Subscriptions
* Wrapping existing REST APIs with GraphQL servers that can be deployed on serverless platforms, and then merging them into a single endpoint
* Triggering serverless functions on database events

In short, we’ll go through how  developers can adopt realtime GraphQL & serverless using Hasura.

-  I would like to present this talk for the June Meetup.
- Length of talk: 30 minutes. 
- https://twitter.com/tanmaigo
- https://twitter.com/tanmaigo/photo

**Comment by paulfioravanti** [comment]
Sounds awesome, Harsha! Thanks for the submission! Consider yourself locked in for the June meetup!

**Comment by sordina** [comment]
Hi @paulfioravanti 

I don't have Twitter, but my LinkedIn, Github, and photo links are:

* https://www.linkedin.com/in/lyndon-maydwell-01455723/
* https://github.com/sordina
* https://raw.githubusercontent.com/sordina/avatars/master/street-1.jpg

**Comment by paulfioravanti** [comment]
Thanks very much for that talk, @sordina, and for sharing your slides!
